packages/components/src/components/card/card.tsx

```diff
@@ -95,7 +95,7 @@ export class Card extends LitElement implements InteractiveComponent {
   /**
    * When `true`, the component is selectable.
    *
-   * @deprecated use `selectionMode` property on a parent `calcite-card-group` instead.
+   * @deprecated in v3.0, removal target v6.0 - Use `selectionMode` property on a parent `calcite-card-group` instead.
```

Comment:
Consistency suggestion:
```suggestion
   * @deprecated in v3.0.0, removal target v6.0.0 - Use the `selectionMode` property on a parent `calcite-card-group` instead.
```